tensorflow/core/kernels/mkl_pooling_ops_common.cc

```diff
@@ -100,7 +100,7 @@ void MklPoolingFwdPrimitive<T>::Setup(const MklPoolingParams& fwdParams) {
     context_.fwd.reset(new pooling_forward(*context_.fwd_pd, *context_.src_mem,
                                            *context_.dst_mem,
                                            *context_.ws_mem));
-#endif
+#endif // ENABLE_MKLDNN_V1
```

Comment:
Nit: (two spaces between `#endif` and `//`.
```suggestion
#endif  // ENABLE_MKLDNN_V1
```

Same for the other two `endif`s.